RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
BTW, there is ~0 extra cost to the existing old implementation by using the step I have suggested via a call to a temporary size=1 EcalDigiCollection.
I agree that it's slightly inefficient and if you do not want to make the change, we can wait for overall change in the interface and modifications.